compiler/tests-num/dune

```diff
@@ -1,12 +1,21 @@
 (executable
  (name main)
+ (modules main test_nats test test_big_ints test_ratios test_nums test_io)
  (libraries num)
  (modes
   js
   (best exe))
  (flags
   (:standard -linkall -w -3-7-33-35-37 -safe-string -no-strict-sequence)))
 
+(library
+ (name test_int31)
+ (modules test_int31)
+ (inline_tests)
```

Comment:
```suggestion
 (inline_tests)
 (enabled_if %{lib-available:qcheck})
```

Reply:
Should I then remove qcheck as a `:with-test` dependency?

Reply:
Keep it for now

Reply:
Looks like the Dune language version is too old. Is it OK to upgrade it? (See last commit)

Reply:
yes